### ISSUE TITLE
Plugin causes Discord to crash when opening Scheduled Events

### ISSUE THREAD

**Issue by joelnedev** [GriefMoDz/better-status-indicators#35]
When clicking on the Events button ([see here](https://user-images.githubusercontent.com/52286831/141852399-a6bb4565-41bc-45e0-a526-974d02189e11.png)) or opening a channel in which an invite to that event was sent (ex. `discord.gg/thisisaninvitecode?event=eventID`), Better Status Indicators causes Discord to crash. The error logged to the DevTools console is `Error: getMaskId(): Unsupported type, size: SIZE_20, status: online, isMobile: false` This behavior is not observed when the plugin is disabled.

**Comment by GriefMoDz** [comment]
Hello, thanks for the bug report! Is there any chance that you could share your settings file with me? You should be able to find it under the "settings" folder of your Powercord install - the file will be called `better-status-indicators.json`. Cheers!

**Comment by joelnedev** [comment]
Of course:
```json
{
  "webShowOnBots": false,
  "enabledModules": [
    "radial-status",
    "status-everywhere"
  ],
  "se-typingStatus": "self+others",
  "themeVariables": false,
  "rs-avatar-inset": 3,
  "rs-hide-speaking-ring": false,
  "se-mobileStatus": "others",
  "seenHardwareAccelerationNotice": true
}
```

**Comment by joelnedev** [comment]
I've also done a little more troubleshooting, and it does not crash when Status Everywhere is disabled.

**Comment by GriefMoDz** [comment]
This should've been resolved as of version 1.13.25 (e73acb2). Let me know if it otherwise isn't and I shall preform some further diagnostics. Cheers!
